spec/lib/rmt/cli/repos_spec.rb

```diff
@@ -15,9 +15,9 @@
     end
 
     context 'without parameters' do
-      let(:argv) { [] }
+      let(:argv) { ['enable'] }
```

Comment:
The context is called `without parameters`, and now there's a parameter `enable` :grimacing:

Reply:
Good point :smile: I will have a look.

Reply:
Well, look again: I added what the test was supposed to do, but did not. It is about testing `rmt-cli repos enable` without any parameters (like an Id)... before my changes, all the test did was calling `rmt-cli repos` ... which was not was the test was all about.